cfme/tests/containers/test_containers_default_services.py

```diff
@@ -36,7 +37,9 @@ def test_containers_default_services():
     sel.wait_until(lambda *args: not sel.is_displayed_text('No Records Found'),
                    'There is no container services at all. ((!) No Records Found)'
                    'Maybe the provider didn\'t load?', timeout=120.0)
-    names = [r[2].text for r in list_tbl.rows()]
+    names = []
+    for _ in paginator.pages():
+        names.extend([r[2].text for r in list_tbl.rows()])
```

Comment:
**OPTIONAL**
rows() returns Row objects that have header names attributized, you could get rid of this magic '2' and reference the column name here instead.